webmin-setup-repo.sh

```diff
@@ -172,6 +180,34 @@ download_key() {
 
 setup_repos() {
   case "$package_type" in
+    rpm-md)
+      if [ "$testing_mode" = "1" ]; then
+        echo "  Setting up Webmin testing repository .."
+        cat << EOF > "$suse_repo_file_testing"
+[webmin-testing]
+name=Webmin Testing
+baseurl=$webmin_download_testing
+enabled=1
+autorefresh=1
+type=rpm-md
+gpgkey=$webmin_key_download
+gpgcheck=1
+EOF
+      else
+        echo "  Setting up Webmin repository .."
+        cat << EOF > "$suse_repo_file"
+[webmin]
+name=Webmin Stable
+baseurl=$webmin_download/download/newkey/yum
+enabled=1
+autorefresh=1
+type=rpm-md
```

Comment:
I think it would be cleaner to abstract the logic here using a few variables specifically needed for Open(SUSE), rather than re-creating the entire repository, since most of the fields are identical.

Also, I believe you should keep `-noarch` in the repository name, as Open(SUSE) supports the concept of noarch packages...